node_lambnik/env-template

```diff
@@ -14,9 +14,10 @@ PROD_TILEGARDEN_USER=
 # Function config information
 ## REQUIRED ##
 LAMBDA_REGION=us-east-1
-LAMBDA_ROLE=role
 
 ## OPTIONAL ##
+# ARN of role associated with this lambda function
```

Comment:
It's worth adding a note here about the overzealous-deletion bug.  Which means we should include an issue number on the claudia.js repo.  Which means we should report the issue.

Reply:
I'm on it, I was just now making sure I could reproduce the not-writing-to-file bug.

Reply:
I've also added a line re: the overzealous deletion to the warning message in `destroy`.